exercises/practice/largest-series-product/.meta/example.ua

```diff
@@ -0,0 +1,3 @@
+Validate       ← |2.2 ⍤ "span must be smaller than string length" ◡(≤⧻) ⍤ "span must not be negative" ≥0 ,
+Digits         ← |1 ⍣(∵⋕) (⍤ "invalid character in digits")
+LargestProduct ← |2 /↥ ≡/× ⍣(◫) [1] : Digits Validate
```

Comment:
I must admit that being able to write this few characters and have it work makes me feel a little bit giddy :)

Reply:
Yes, I feel you. This lets me feel like I am in the matrix